examples/info.rs

```diff
@@ -25,7 +25,18 @@ fn main() {
 
     let system = System::new(url);
 
-    let version = system.info.get_version();
-
-    println!("{:?}", version);
+    let get_version_result = system.info.get_version();
+
+    match get_version_result {
+        Ok(version) => println!("Success: {:?}", version),
+        Err(error) => match error {
+            RequestError::MavErr(mav_err) => match mav_err {
+                info::InfoError::Unknown(s) => println!("Unknown MAVLink error ({:?})", s),
+                info::InfoError::InformationNotReceivedYet(s) => {
+                    println!("Information not received yet ({:?})", s)
+                }
+            },
+            RequestError::RpcErr(rpc_err) => println!("RPC error: {:?}", rpc_err),
+        },
+    }
 }
```

Comment:
And there is example of use

Reply:
Is it possible, in Rust, to write directly something like the following?

```rust
    match get_version_result {
        Ok(version) => println!("Success: {:?}", version),
        Err(RequestError::MavErr(mav_err) => match mav_err => { ... },
        Err(RequestError::RpcErr(rpc_err) => ...
    }
```

Reply:
> Is it possible, in Rust?

Unfortunately no.
Matching should go "down the tree".
But this allow to implement good structured error processing.

Reply:
But it possible to:
```rust

    match get_version_result {
        Ok(version) => println!("Success: {:?}", version),
        Err(error) => {
            if let RequestError::MavErr(mav_err) = error {
                println!("MAVLink Error: {:?}", mav_err)
            } else {
                println!("Other error: {:?}", error)
            }
        }
    }
```

Reply:
Or 
```rust
Err(ref error) => {
   if let RequestError::MavErr(mav_err) = error {
        println!("MAVLink Error: {:?}", mav_err)
    }
    if let RequestError::RpcErr(rpc_err) = error {
        println!("RPC error: {:?}", rpc_err)
   }
}
```

Reply:
So my first answer was wrong. It is possible :)